src/api/routers/participants/participantsRouter.ts

```diff
@@ -127,23 +125,7 @@ export function createParticipantsRouter() {
 
   participantsRouter.get('/approved', isApproverCheck, async (req, res) => {
     const participants = await getParticipantsApproved();
-
-    const sitesList = await getSiteList();
-    const siteMap = new Map<number, AdminSiteDTO>(sitesList.map((s) => [s.id, s]));
-
-    const allParticipantTypes = await ParticipantType.query();
-    const result = participants
-      .map((p) => {
-        const currentSite = p?.siteId === undefined ? undefined : siteMap.get(p.siteId);
-        return {
-          ...p,
-          types: mapClientTypeToParticipantType(
-            currentSite?.clientTypes || [],
-            allParticipantTypes
-          ),
-        };
-      })
-      .sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
This function was pulling data from AdminDB, but we want to portal DB  info instead.  This made the code much simpler and also removed several imports.